src/FffGcodeWriter.h

```diff
@@ -398,6 +398,16 @@ class FffGcodeWriter : public SettingsMessenger, NoCopy
      */
     void processInsets(const SliceDataStorage& storage, LayerPlan& gcodeLayer, const SliceMeshStorage* mesh, const PathConfigStorage::MeshPathConfigs& mesh_config, const SliceLayerPart& part, unsigned int layer_nr, EZSeamType z_seam_type, Point z_seam_pos) const;
     
+    /*!
+     * Generate the a spiralized wall for a given layer part.
+     * \param[in] storage where the slice data is stored.
+     * \param gcodeLayer The initial planning of the gcode of the layer.
```

Comment:
We generally add the specification if a function parameter is an output value.
Please make this line read
```
     * \param[out] gcodeLayer The initial planning of the gcode of the layer.
```